docs/docsite/rst/user_guide/playbooks_filters.rst

```diff
@@ -1349,6 +1349,30 @@ Hash type 'blowfish' (BCrypt) provides the facility to specify the version of th
 
 .. versionadded:: 2.12
 
+You can also use Ansible Vault compatible filter to encrypt data::
```

Comment:
Wordsmithing and adding a link here. We should definitely link to the more comprehensive docs on using vault, if we're going to do this.
```suggestion
You can also use the Ansible :ref:`vault <vault>` filter to encrypt data::
```